lascli/parser/models.py

```diff
@@ -60,11 +58,11 @@ def update_data_bundle(las_client: Client, model_id, data_bundle_id, **optional_
     return las_client.update_data_bundle(model_id, data_bundle_id, **optional_args)
 
 
-def create_training(las_client: Client, model_id, data_bundle_ids, instance_type, **optional_args):
+def create_training(las_client: Client, model_id, data_bundle_ids, data_scientist_assistance, **optional_args):
     return las_client.create_training(
         model_id=model_id,
         data_bundle_ids=data_bundle_ids,
-        instance_type=instance_type,
+        data_scientist_assistance=data_scientist_assistance or None,
```

Comment:
If data_scientist_assistance is false, then this will evaluate to None. I don't think that is what we want.

Reply:
What would you expect the behavior to be if you don't provide this flag? I think not providing it should mean that we use the API default (which setting to None achieves)